libs/banklessdao/content-gateway-sdk/src/client/ContentGatewayClient.ts

```diff
@@ -55,17 +48,11 @@ export type ContentGatewayClientV1 = {
     ) => TE.TaskEither<ClientError, Record<string, unknown>>;
     /**
      * Tries to save the supplied [[payload]] to the Content Gateway API.
-     * The schema of the payload must be {@link ContentGatewayClientV1#register | register}ed
+     * The schema of the payload must be {@link ContentGatewayClient#register | register}ed
      * with the Content Gateway API first.
      */
     save: <T>(
-        saveParams: SaveParams<T>
-    ) => TE.TaskEither<ClientError, Record<string, unknown>>;
-    /**
-     * Same as {@link ContentGatewayClientV1#save} but sends multiple entries at once.
-     */
-    saveBatch: <T>(
-        saveBatchParams: SaveBatchParams<T>
+        payload: SaveDataParams<T>
```

Comment:
There was no point in having both a `save` and a `saveBatch`. If you want to save one record you just send an array with one item. This was removed across the board.